webpack/assets/javascripts/react_app/components/notifications/toast/Toast.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Icon from '../../common/Icon';
+import Alert from '../../common/Alert';
+
+const Toast = (props) => {
```

Comment:
You can always break the statement into 2 or more lines. The question is whether or not destructuring the props in this case adds clarity or causes confusion.